Makefile

```
@@ -3,6 +3,9 @@ PYTHON ?= python3
 .PHONY: all
 all: style-check type-check test
 
+.PHONY: all-checks
+all-checks: style-check type-check doclinter
```

```suggestion
.PHONY: lint
lint: style-check type-check doclinter
```

Let reduce the name of the rules as much as possible!

---

I guess you can get into semantics, of whether type-checking is linting 😅

But yeh no worries, I just want something that can replace how I'd usually use `pre-commit` in my other projects (without getting into that debate for now 😉)

---

Actually, it's a completely personal request since I use `make lint` in all my projects when it comes to "style/formatter/whatever prettifier" (and thus it's easier to type!)

---

Well another name could have been `check` which now.. maybe is best suited ?

---

Ok, changed to `check`, final offer 😄